povm_toolbox/post_processor/median_of_means.py

```diff
@@ -76,6 +76,7 @@ def __init__(
         self,
         povm_sample: POVMPubResult,
         dual: BaseDual | None = None,
+        combine_counts: bool = False,
         *,
```

Comment:
```suggestion
        *,
        combine_counts: bool = False,
```
This should be a keyword-only argument

Reply:
The same has to be done elsewhere.